content/kubeone/master/architecture/requirements/provider_support_matrix/azure/azure.en.md

```diff
@@ -0,0 +1,79 @@
++++
+title = "Azure"
+date = 2019-05-19T12:07:15+02:00
+weight = 7
+
++++
+
+## Prepare Azure Environment
+
+For provisioning Kubernetes clusters with the [Azure cloud provider](https://github.com/kubermatic/machine-controller/tree/master/pkg/cloudprovider/provider/azure) Kubermatic Kubernetes Platform (KKP) needs a service account. Please follow the following steps steps to create a matching service account and the roles:
+
+### Login to Azure and Get Basic Information
+
+Login to Azure with [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest) `az`.
+
+```bash
+az login
+```
+
+This command will open in your default browser a window where you can authenticate. After you successful logged in get your subscription ID.
+
+```bash
+az account show --query id -o json
+
+********-****-****-****-************
+```
```

Comment:
Can we split the command and output into two blocks? That way users can more easily copy the command (e.g. by using the copy button in the right corner of the block).

Reply:
I think we do not need the output at all